corehq/apps/registration/static/registration/js/new_user.ko.js

```diff
@@ -11,6 +11,7 @@ hqDefine('registration/js/new_user.ko', [
     'jquery-ui/ui/effects/effect-slide',
     'intl-tel-input/build/js/intlTelInput.min',
     'hqwebapp/js/password_validators.ko',
+    'hqwebapp/js/captcha', // shows captcha
```

Comment:
do we need that comment? i think the file name is self explanatory

Reply:
removed